app/services/csv/sales_log_csv_service.rb

```diff
@@ -321,6 +322,21 @@ def order_address_fields_for_support(attributes)
       end
     end
 
+    # as part of CLDC-3719 it was decided to move the saledate question to be first in the form
+    # this caused issues reported in CLDC-4025 where the user only enter saledates for their active organisation
+    # we decided to move the organisation question back to being first
+    # however, we did not want to reorder the CSV export as this would disrupt existing users' data pipelines
+    # so, this function reorders questions back when exporting CSVs
+    # next year, we can remove this function as we will be reordering the csv fields anyway
+    def order_saledate_question_before_owning_organisation_question(ordered_questions)
```

Comment:
happy to hear thoughts on where best to place this code, did consider making this a form method instead

Reply:
I think this makes the most sense. It only applies to the csv export so it doesn't need to be on the form itself.